src/amo/components/AddonRecommendations/index.js

```diff
@@ -67,6 +68,12 @@ export class AddonRecommendationsBase extends React.Component<Props> {
       cookie.save(TAAR_COHORT_COOKIE_NAME, this.cohort, { path: '/' });
     }
 
+    // Set a custom dimension for the cohort.
```

Comment:
Not sure this comment is super useful since the code is pretty straightforward. Maybe we would like to know why we do this instead, what do you think? (or no comment at all)

Reply:
Fair enough. I think I'll just remove it.